src/include/sof/schedule.h

```diff
@@ -70,6 +70,7 @@ struct sof;
 
 #define TASK_PRI_IPC	1
 #define TASK_PRI_IDC	1
+#define TASK_IDLE	20
```

Comment:
@mrajwa lets keep the PRI prefix if this is meant for priority?

Reply:
@ranj063, the point is idle tasks don't own priorities. So calling them this way would be misleading i think.

Reply:
It looks like a priority, so `TASK_PRI_IDLE` would be more consistent. Could be #defined to a number < LOW to keep them ordered.